app/plugins/file-log.md

```diff
@@ -38,6 +43,7 @@ form parameter                | default | description
 `name`                        |         | The name of the plugin to use, in this case: `file-log`
 `consumer_id`<br>*optional*   |         | The CONSUMER ID that this plugin configuration will target. This value can only be used if [authentication has been enabled][faq-authentication] so that the system can identify the user making the request.
 `config.path`                 |         | The file path of the output log file. The plugin will create the file if it doesn't exist yet. Make sure Kong has write permissions to this file.
+`reopen`                      | `false` | Determines whether the log file is closed and reopened on every request. If the file is not reopened, and has been removed/rotated, the plugin will keep writing to the stale file descriptor, and hence loose information.
```

Comment:
'lose' not 'loose' (i would fix and merge this myself, but the above comment needs a bit more fleshing out i think? i can handle it if you like).